tests/mock_tables/global_db/state_db.json

```diff
@@ -123,5 +123,32 @@
   "TEMPERATURE_INFO|thermal1": {
     "temperature": "20.5",
     "is_replaceable": "False"
+  },
+  "BUFFER_MAX_PARAM_TABLE|Ethernet0": {
```

Comment:
state_db in global db will not have interface related configuration

Reply:
Moved to asics db